zyte_api/aio/errors.py

```diff
@@ -23,4 +26,5 @@ def parsed(self):
 
     def __str__(self):
         return f"RequestError: {self.status}, message={self.message}, " \
```

Comment:
it seems request_id is going to be included twice here: in self.message, and in self.request_id as well.